### PR TITLE
[docs]: Pagination Documentation

### PR DESCRIPTION
**Pull request by shivangT**
-Created missing pagination doc
-Updated new page layout

### CODE REVIEW

File: docs/docs/widgets/pagination.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+id: pagination
+title: Pagination
+---
+# Pagination
+
+Pagination enables the user to select a specific page from a range of pages.
+
+:::tip
+It is used seperate content into discrete pages.
+:::
```
Comment (withshubh) [inline docs/docs/widgets/pagination.md:11]: `It is used seperate content into discrete pages` can be moved in the description and inside tip maybe we can add something like `You can club Pagination widget with the List View widget.` Or any other use case if you have in your mind.

File: docs/docs/widgets/pagination.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - Widget Reference - Code editor](/img/widgets/pagination/pagination.png)
+
+</div>
```
Comment (withshubh) [inline docs/docs/widgets/pagination.md:17]: Let's use the `` tags. We don't need to use markdown wrapped inside div anymore.

File: docs/docs/widgets/pagination.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+mouse pointer over the widget.
+
+Under the <b>General</b> accordion, you can set the value in the string format. 
+Now hovering over the widget will display the string as the tooltip.
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - widget- button](/img/tooltip.png)
+
+</div>
```
Comment (withshubh) [inline docs/docs/widgets/pagination.md:47]: Replace with ``

File: docs/sidebars.js [hunk 4]
```diff
@@ -146,7 +146,8 @@ const sidebars = {
         'widgets/timer',
         'widgets/toggle-switch',
         'widgets/tree-select',
-        'widgets/vertical-divider'
+        'widgets/vertical-divider',
+        'widgets/pagination'
```
Comment (withshubh) [inline docs/sidebars.js:150]: All the widgets and datasources in the sidebar are in lexicographic order.
Please place pagination accordingly.

**Review by withshubh — CHANGES_REQUESTED** [review]
Please make the suggested changes.
The rest looks good.

File: docs/docs/widgets/pagination.md (hunk 3 above)
Comment (shivangT) [reply docs/docs/widgets/pagination.md:47]: Updating  tag here with outer div, as this image gets left aligned if div not provided.

**Comment by shivangT** [comment]
Hey @withshubh 
Reverted to the comments. Please approve if good to go

**Comment by shivangT** [comment]
Hey @Navaneeth-pk 
The changes are reverted. 
Can we close this PR?

**Review by withshubh — APPROVED** [review]

**Pull request merged**